Jenkinsfile

```diff
@@ -47,6 +47,8 @@ pipeline
                         {
                                     sh 'npm run version-stamp'
                                     sh 'npm run version'
+                                    sh 'npm rebuild canvas --update-binary'
```

Comment:
@SaharKargar 
Can you rebase your changes? 
There are a few changes that is not related to data-* work.

Reply:
@jingteng25742 
Rebased and pushed.